doc/user/content/integrations/cube.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Cube"
+description: "How to connect a Cube deployment to Materialize"
+aliases:
+  /third-party/cube/
+menu:
+  main:
+    parent: "integration-guides"
+    name: "Cube"
+---
+
+[Cube](https://www.cubejs.com/) is an open-source **Headless BI platform**. This integration guide will teach you how to connect Cube with Materialize in a few steps using the browser.
+
+## Pre-requisite
+
+Before following the steps, make sure to meet the next items:
+
+- Have a Cube account. Create [one here](https://cubecloud.dev/auth/signup).
+- A Materialize's region enabled.
+
+## Steps
```

Comment:
Our guides are getting a little wild in terms of not having a consistent structure. As much as possible, let's try to follow a more "actionable" structure similar to the [dbt guide](https://materialize.com/docs/integrations/dbt/), or the [PostgreSQL CDC guide](https://materialize.com/docs/integrations/cdc-postgres/). This should make it easier for users to find what they're looking for (though this is a short guide)!

Reply:
For example:

## Create and configure a Cube deployment

1. Login to Cube Cloud and create a deployment. To improve latency, make sure that you use the same regions as the Materialize region you want to connect to.

2. Set up ...

## Connect to Materialize

1. Configure the connection ...

2. Build the schema ...

...

## Related pages

...